cypress/integration/FieldSetField/can_be_required.feature

```diff
@@ -0,0 +1,5 @@
+Feature: Required status for the FieldSetField
+
+    Scenario: Rendering a FieldSetField that is required
+        Given a FieldSetField with label and a required flag is rendered
```

Comment:
Technically this is a FieldSetField with a legend. I'd rename it to legend to prevent confusion.

Reply:
0863986

It's a technically a label now, but it behaves as a legend.